chaosk8s/node/actions.py

```diff
@@ -23,7 +23,7 @@ def _select_nodes(name: str = None, label_selector: str = None,
                   count: int = None, secrets: Secrets = None,
                   pod_label_selector: str = None,
                   pod_namespace: str = None, first: bool =
-                  False) -> list[client.V1Node:]:
+                  False) -> List[client.V1Node:]:
```

Comment:
One small typo still. Remove the `:` before `]` :)

Also, you are missing the import for List at the top of your module:

```
from typing import Any, Dict
```

to

```
from typing import Any, Dict, List
```